test/rekt/features/broker/control_plane.go

```diff
@@ -309,6 +372,9 @@ func readyBrokerIsAddressable(ctx context.Context, t feature.T) {
 func brokerClassIsImmutable(ctx context.Context, t feature.T) {
 	broker := getBroker(ctx, t)
 
+	if broker.Annotations == nil {
+		broker.Annotations = map[string]string{}
```

Comment:
Nit, since it use it right below.
```suggestion
		broker.Annotations = make(map[string]string, 1)
```

Reply:
meh, for a test, it does not matter much